### ISSUE TITLE
clicking JFT copyright link opens in app

### ISSUE THREAD

**Issue by pjaudiomv** [paulnagle/NA-Ireland-Ionic-2#40]
just giving you a heads up here as well, this is kind of an edge case but. clicking the copyright link from the Just For Today opens inside the app, then your kind of stuck in the app with no way back. You then have to close the app and re-open. I fixed this by adding some css to `justfortoday.scss` this essentially makes the link not usable and appears as normal text. theres probably a better way to fix this like adding on click event or removing href tag but thats what I did.

```
 a {
   pointer-events: none;
   cursor: default;
   text-decoration: none;
   color: black;
 }
```

**Comment by pjaudiomv** [comment]
yes this is a VERY bad way to do it, this killed the tabs at the bottom so the style sheet must be loaded for whole project. either got to set more specificity or do it differently.

**Comment by pjaudiomv** [comment]
this ended up working
```
tr td a {
   pointer-events: none;
   cursor: default;
   text-decoration: none;
   color: black;
 }
```

**Comment by paulnagle** [comment]
Thanks Patrick, will include this whenever I next update the app
